### ISSUE TITLE
Skip button isn't working with music

### ISSUE THREAD

**Issue by joshualyon** [iamcanadian2222/ST-Kodi#2]
> This is what I'm getting with the play and skip buttons.... the others seem to be working fine. I've double checked my login and IP. I have Yatse running fine on 4 other tablets so I think my Kodi is set up right. Let me know if I can try anything.

```
844b5940-191f-4354-b498-996624076741 8:16:51 PM: debug running todos
844b5940-191f-4354-b498-996624076741 8:16:51 PM: debug Last Command: Input.Right
844b5940-191f-4354-b498-996624076741 8:16:51 PM: debug [id:1, result:OK, jsonrpc:2.0]
844b5940-191f-4354-b498-996624076741 8:16:51 PM: debug HEADER: [content-length:38, http/1.1 200 ok:null, cache-control:private, max-age=0, no-cache, connection:Keep-Alive, accept-ranges:none, content-type:application/json, date:Wed, 10 Feb 2016 01:11:14 GMT]
844b5940-191f-4354-b498-996624076741 8:16:51 PM: trace Response Received (with Headers and Body)
844b5940-191f-4354-b498-996624076741 8:16:49 PM: trace Sending command: Input.Right
844b5940-191f-4354-b498-996624076741 8:16:29 PM: debug running todos
844b5940-191f-4354-b498-996624076741 8:16:29 PM: debug Last Command: Player.GoTo
844b5940-191f-4354-b498-996624076741 8:16:29 PM: debug [id:1, error:[message:Failed to execute method., code:-32100], jsonrpc:2.0]
844b5940-191f-4354-b498-996624076741 8:16:29 PM: debug HEADER: [content-length:86, http/1.1 200 ok:null, cache-control:private, max-age=0, no-cache, connection:Keep-Alive, accept-ranges:none, content-type:application/json, date:Wed, 10 Feb 2016 01:10:53 GMT]
844b5940-191f-4354-b498-996624076741 8:16:29 PM: trace Response Received (with Headers and Body)
844b5940-191f-4354-b498-996624076741 8:16:29 PM: trace Sending command: Player.GoTo
844b5940-191f-4354-b498-996624076741 8:16:29 PM: debug Executing 'nextTrack'
844b5940-191f-4354-b498-996624076741 8:16:07 PM: debug running todos
844b5940-191f-4354-b498-996624076741 8:16:07 PM: trace UPnP Response
844b5940-191f-4354-b498-996624076741 8:16:07 PM: debug HEADER: [nts:upnp:propchange, nt:upnp:event, content-length:369, sid:uuid:0f0ff704-cadf-ef27-c0e5-f3236983c673, host:192.168.1.6:39500, seq:6, user-agent:Neptune/1.1.3, content-type:text/xml; charset="utf-8", notify /notify http/1.1:null]
844b5940-191f-4354-b498-996624076741 8:16:07 PM: trace Response Received (with Headers and Body)
844b5940-191f-4354-b498-996624076741 8:16:06 PM: debug running todos
844b5940-191f-4354-b498-996624076741 8:16:05 PM: debug Current playing title is: 25 Or 6 To 4
844b5940-191f-4354-b498-996624076741 8:16:05 PM: debug METADATA: 25 Or 6 To 4ChicagoChicagoChicagoChicagoUnknownChicago IX - Greatest HitsClassic Rockhttp://192.168.1.101:1455/thumb?path=image%3A%2F%2FD%253a%255cMusic%255cChicago%255cChicago%2520-%2520Chicago%2527s%2520Greatest%2520Hits%255cFolder.jpg%2F12016-02-07T20:50:36+05:004600.0object.item.audioItem.musicTrack
844b5940-191f-4354-b498-996624076741 8:16:05 PM: debug WE HAVE METADATA
```

> Sending command: Player.GoTo
> 8:16:29 PM: debug [id:1, error:[message:**Failed to execute method.**, code:-32100], jsonrpc:2.0]

**Comment by joshualyon** [comment]
Reference Post:
https://community.smartthings.com/t/release-kodi-for-smartthings-beta/38305/47?u=joshua_lyon

**Comment by joshualyon** [comment]
The player ID was not getting set when doing UPnP parsing of the track information. As such, when the next track command was being issued, it was being issued with the default video player id of `1` (instead of the audio player id `0`)

Fixed in 9bb4f0c
